src/main/resources/application.properties

```diff
@@ -0,0 +1,69 @@
+#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+#spring.datasource.url=jdbc:mysql://ps3.bullhorncloud.com/snisandbox
+#spring.datasource.username=bullhorntestprod
+#spring.datasource.password=profsvcs1
+
+## DO NOT CHANGE THIS! used in Unit tests to make sure that test scripts are only run when the testing profile is used
+profilename=staging
+
+## Core Database
+jdbc.driverClassName=com.mysql.cj.jdbc.Driver
+jdbc.url=jdbc:mysql://ps3.bullhorncloud.com/snisandbox?useSSL=false
+jdbc.user=snisandbox
+jdbc.password=ajdiH3Tser3s90j32s
+
+## Log Database
+jdbc.logdriverClassName=com.mysql.cj.jdbc.Driver
+jdbc.logdburl=jdbc:mysql://ps2.bullhorncloud.com/bullhornlog
+jdbc.logdbusername=bullhornlog
+jdbc.logdbpassword=profsvcs1
+
+apiurl=https://api-west9.bullhornstaffing.com/webservices-2.5/?wsdl
+apiusername=
+apipassword=
+apikey=24
+apiUserID=1
+
+corporationID=
+privateLabelID=
+
+applicationRoot=
+
+numEventThreads=5
+numEventsPerBatch=100
+applicationDateFormat=MM/dd/yyyy
+applicationTimeFormat=HH:mm
+standardSubscriptionName=
+standardCronExpression=0 */5 * * * ?
+
+date.last.modified.cron.expression=0 0/1 * 1/1 * ? *
+
+metaCacheMinutes=1440
+
+## Email
+mail.host=smtp.socketlabs.com
+mail.port=465
+mail.username=server36313
+mail.password=m2Q6RxEo7p9C4Ykn5SW
+mail.sender=notifications@bullhorncloud.com
+mail.senderName=Notifications
+mail.disabled=false
+mail.routeToWhenDisabled=
+
+## BH REST
+rest.clientId=b627302e-e971-4a8b-b122-91c622f6bf39
+rest.clientSecret=hfP1labCstTMs1CGipakgWYJ627KHr0y
+rest.sessionMinutesToLive=1400
+
+## Hibernate configs
+hibernate.showSql=true
+hibernate.generateDdl=true
+
+logEntries.token=
+
+orca.username=
+orca.password=
+
+rest.prefix.core=/main/resttrigger
```

Comment:
This is going to be used as the prefix for the rest trigger endpoints. Configured in the `@Controller` annotation. I believe this replaces everything we were doing before with the main-servlet.xml and custom-servlet.xml file